test/common/build/controllers.go

```diff
@@ -653,6 +649,10 @@ func RunBuildRunningPodDeleteTest(t testingT, clusterAdminClient *client.Client,
 	if newBuild.Status.Phase != buildapi.BuildPhaseError {
 		t.Fatalf("expected build status to be marked error, but was marked %s", newBuild.Status.Phase)
 	}
+
+	// allow time for the failed event to get created
+	time.Sleep(5 * time.Second)
```

Comment:
Is that a flake waiting to happen?

Also, above in this file:
```
	// give the poddeletecontroller's build cache time to be updated with the build object
	// so it doesn't get a miss when looking up the build while processing the pod delete event.
	time.Sleep(10 * time.Second)
```

Is the above one straight removable now the poddeletecontroller has gone?